src/sdk.ts

```diff
@@ -94,10 +93,15 @@ async function getOneLineStdout(program: string, args: string[]): Promise<[strin
 }
 
 export async function reinstallMxpy(version: Version) {
-    let mxpyUp = storage.getPathTo("mxpy-up.py");
-    await downloadFile(mxpyUp, MxpyUpUrl);
+    const mxpyUp = storage.getPathTo("mxpy-up.py");
+    const mxpyUpUrl = getMxpyUpUrl(version);
+    await downloadFile(mxpyUp, mxpyUpUrl);
+
+    const isV6OrNewer = version.isNewerOrSameAs(Version.parse("6.0.0-alpha.0"));
```

Comment:
is `6.0.0-alpha.0` a real tag? 
the https://raw.githubusercontent.com/multiversx/mx-sdk-py-cli/6.0.0-alpha.0/mxpy-up.py URL composed at line 118 does not work

Reply:
Not a real tag, it's the "smallest possible semver for v6". Here, we could have not used `6.0.0` for comparison. If we did so, we could have not installed, alpha / beta versions of mxpy.

Because, according to semver, `6.0.0-alpha.0 < 6.0.0-any-other-pre-release.42 < 6.0.0`.